ckanext/validation_schema_generator/templates/base.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block scripts %}
+    {{ super() }}
+
+    {% set type = 'asset' if h.ckan_version().split('.')[1] | int >= 9 else 'resource' %}
```

Comment:
This would break if the CKAN major version were updated (eg CKAN 3). It would be better to incorporate a call to `toolkit.check_ckan_version`